tests/fixtures/music_nerd_pro/combination_responses_with_history_direct.hocon

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+# This file defines everything necessary for a data-driven test.
+{
+    # Describes what agent to test against.
+    "agent": "music_nerd_pro",
+
+    "connections": ["direct"]
+
+    # Interactions are a series of dictionaries with request elements paired with
+    # descriptions of response checks.
+    "interactions": [
+        {
+            # This is what we send as input to streaming_chat()
+            "text": "Who did Yellow Submarine?",
+
+            # The response block treats how we are going to test what comes back
+            "response": {
+
+                # Text block says how we are going to examine the text of the response.
+                "text": {
+
+                    # Keywords says we are going to look for exact matches for each
+                    # element in a list of strings.  All elements need to show up
+                    # in order to pass.
+                    "keywords": ["Beatles", "\"running_cost"\": 3.0]
+                }
+
+
+            
```

Comment:
Feel free to get rid of all this extra whitespace at your leisure. Here and other files.